### ISSUE TITLE
Amazon Music

### ISSUE THREAD

**Issue by NovaGL** [simple-icons/simple-icons#3163]
So Amazon Music is very confusing branding wise, just read this site and try and workout what the name of the service is.

[Amazon Music sub-brands](https://www.amazon.com/gp/help/customer/display.html?nodeId=GW3PHAUCZM8L7W9L)

I'm happy with the below image but I don't know what branding link to use.

![amazonmusic](https://user-images.githubusercontent.com/16315601/83749106-87b83600-a6a6-11ea-99be-4a47bf60f8f8.png)

**Name:** Amazon Music
**Website:** https://music.amazon.com/
**Official resources for icon and color:** https://press.aboutamazon.com/images-videos

**Comment by NovaGL** [comment]
Yeh. Do a Google image search for Amazon music and it has various different options. I chose that one because it's on the wiki page. Not that it means it's correct but it's the only documentation I could find.

**Comment by PeterShaggyNoble** [comment]
Yes, given that there are no official guidelines and that, while being no means definitive, Wikipedia uses the "Music" version, I'd be happy enough for us to go with that too. Also, according to [Logopedia](https://logos.fandom.com/wiki/Amazon_Music), which also wouldn't be definitive, the "Amazon Music" version is considered to be secondary.

I found [this SVG](https://images-na.ssl-images-amazon.com/images/G/01/digital/music/player/web/AmazonMusicGradientLogo._CB509440606_.svg) in the source of https://music.amazon.com/, if it's any help to you.

Still no clearer on whether we should go with the purple, "Prime Blue" or "Squid Ink" for the colour but, at a push, based on what I said above and it's colouring on Logopedia, I'd lean towards the purple.

**Comment by NovaGL** [comment]
Good job on finding the icon. That's why I chose the blue color even though the latest one shows purple I don't believe that's a good long term fit.

If we look in the Sonos app it's blue.
![image](https://user-images.githubusercontent.com/16315601/83935129-d4f8ec80-a7f9-11ea-99ae-62d7386298d4.png)

If I was pushed to choose one color, even though I prefer "Prime Blue" I would go with "Squid Ink" which seems to be the global color used for "prime" brands.

**Comment by PeterShaggyNoble** [comment]
Yep, it does seem to me like "Prime Blue" is the brand colour of the parent Prime brand, while "Squid Ink" is the brand colour of its sub-brands. Based on that and in the absence of anything saying otherwise, I'd be happy to go with it here, despite _personally_ thinking purple is the correct choice. So, to sum up, we're going to go with:
1. Just the music wordmark for the icon, and,
2. "Squid Ink" for the colour.

**Comment by NovaGL** [comment]
Any of the @simple-icons/maintainers have suggestions on how this icon can be included?

**Comment by adamrusted** [comment]
> Any of the @simple-icons/maintainers have suggestions on how this icon can be included?

I agree with @PeterShaggyNoble, the most universal version I'm seeing is the wordmark (with the arrow underneath) in the purple.

**Comment by adamrusted** [comment]
> > Any of the @simple-icons/maintainers have suggestions on how this icon can be included?


I agree with @PeterShaggyNoble, the most universal version I'm seeing is the wordmark (with the arrow underneath) in the purple.

I'd go with the homepage of Amazon Music (where the SVG was extracted) as the source, and include other links (for color reference etc) to aid those looking at the icon for any reviews (and to help toward #2846)

**Comment by PeterShaggyNoble** [comment]
Looks like we're in agreement, so 🙂 I think this is ready to take to PR now, @NovaGL.

**Comment by RiteshK-611** [comment]
Amazon Music updated their Icon

![](https://d5fx445wy2wpk.cloudfront.net/icons/amznMusic_favicon.png)

**Comment by PratyushJayachandran** [comment]
This seems to be the branding page for amazon music
https://artists.amazonmusic.com/tools
So which one seems appropriate?

The bargraph logo seems more like a favicon thing. The appstore icon atleast for now is still the word music with smile underneath it.

**Comment by PeterShaggyNoble** [comment]
And, so, another PR bites the dust! 😆

Are we destined for all eternity to be haunted by this, on the surface, seemingly simple icon that Amazon are making infuriatingly difficult?!

Who's turn is it to have a go at it next?
